test/parallel/test-dgram-createSocket-type.js

```diff
@@ -52,10 +52,12 @@ validTypes.forEach((validType) => {
     // note: linux will double the buffer size
     assert.ok(socket.getRecvBufferSize() === 10000 ||
               socket.getRecvBufferSize() === 20000,
-              'SO_RCVBUF not 1300 or 2600');
+              'SO_RCVBUF not 10000 or 20000, ' +
+              `was ${socket.getRecvBufferSize()}`);
```

Comment:
Non blocking nit: please indent this code two more so it is possible to distinguish it from the other lines. The same applies to the part below.